histomicstk/segmentation/label/trace_boundary.py

```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import ctypes
-from .isbf import isbf
+from .trace_object_boundary import trace_object_boundary
 
 
 def trace_boundary(Mask, Connectivity=4, XStart=None, YStart=None,
```

Comment:
I think this is a great place to pickup speed - just pre-calculate 3 rotations at the beginning (90, 180, 270) and use these in subsequent calculations.